lib/fixtures/LICENSE.md

```diff
@@ -1,6 +1,6 @@
 #Learn.co Educational Content License
 
-Copyright (c) 2015 Flatiron School, Inc
```

Comment:
Hm, I'm not actually sure. That's a good question.  This suggests that it doesn't actually matter...http://www.contentious.com/2007/01/07/copyright-notice-is-the-year-really-necessary/.